plugins/repository-s3/src/internalClusterTest/java/org/opensearch/repositories/s3/S3BlobStoreRepositoryTests.java

```diff
@@ -208,14 +204,12 @@ private void validateAuthHeader(HttpExchange exchange) {
             final String authorizationHeaderV4 = exchange.getRequestHeaders().getFirst("Authorization");
             final String authorizationHeaderV3 = exchange.getRequestHeaders().getFirst("X-amzn-authorization");
 
-            if ("AWS3SignerType".equals(signerOverride)) {
-                assertThat(authorizationHeaderV3, startsWith("AWS3"));
-            } else if ("AWS4SignerType".equals(signerOverride)) {
+            if ("AWS4SignerType".equals(signerOverride)) {
                 assertThat(authorizationHeaderV4, containsString("aws4_request"));
             }
-            if (region != null && authorizationHeaderV4 != null) {
-                assertThat(authorizationHeaderV4, containsString("/" + region + "/s3/"));
-            }
+            // if (region != null && authorizationHeaderV4 != null) {
+            // assertThat(authorizationHeaderV4, containsString("/" + region + "/s3/"));
+            // }
```

Comment:
I think we should put this back to make sure the request is scoped to the right region in the tests.

Reply:
Uncommented this.